corehq/apps/enterprise/enterprise.py

```diff
@@ -17,15 +17,14 @@
 from corehq.apps.domain.models import Domain
 from corehq.apps.es import forms as form_es
 from corehq.apps.export.dbaccessors import ODataExportFetcher
-from corehq.apps.reports.filters.users import \
-    ExpandedMobileWorkerFilter as EMWF
 from corehq.apps.users.dbaccessors import (
     get_all_user_rows,
     get_mobile_user_count,
     get_web_user_count,
 )
 from corehq.apps.users.models import CouchUser, Invitation
 from corehq.util.quickcache import quickcache
+from corehq.apps.es.users import UserES
```

Comment:
nit: these should be alphabetical so it should be up with the other `es` import

Reply:
Thank you Cal, fixed here: 472928798a8a327a0048e22a1dc5b048a922e09a